src/Samples/DIPS.Xamarin.UI.Samples/Extensions/PropertyChangedViewModel.cs

```diff
@@ -21,7 +21,13 @@ internal class PropertyChangedViewModel : INotifyPropertyChanged
         public string MyFirstProperty
         {
             get => m_myFirstProperty;
-            set => this.Set(ref m_myFirstProperty, value, PropertyChanged);
+            set
+            {
+                this.Set(ref m_myFirstProperty, value, PropertyChanged);
+
+                //Alternate fluent version
+                this.On(PropertyChanged).After(ref m_myFirstProperty, value);

```

Comment:
After doesn't quite feel right

Reply:
I agree. Do you anything else in mind?

Reply:
Brainstorming:
`this.On(PropertyChanged).With(ref m_prop, value).Raise(propname(callermember))`
`this.Raise(ref m_prop, value).On(PropertyChanged)`
🤔

Reply:
The extra `.Raise` on top of `On().With()` kind of doesnt make sense.
This is due to us wanting to stick with `On` when it actually should have been `Raise`. 
I kind of read that first solution as : "Rais on property changed with backing store has new value and and raise property changed on my property". 

Second solution :
`Raise` should raise the event, and not set values IMO.
I read that solution as : "Raise backing store with new value raise on property changed"

I dont feel like any of this feels fluent. 

With `And()` I read it:
"Raise On Property changed and set my backing store with new value"
So, if we had used `Raise instead of `On` this would have felt more fluent.